app.py

```diff
@@ -264,5 +266,58 @@ def get_pref(topic):
     )
 
 
+@app.route(f"{URL_PREFIX}/onboarding", methods=["GET", "POST"])
+@auth.requires_login
+def onboarding_survey():
+    if auth.get_account_status() != "pending_onboarding_survey":
+        return redirect(url_for("home"))
+
+    if request.method == "POST":
+        print(request.form)
+
+        with DB_ENGINE.connect() as conn:
+            repo = DbDemographicsRepository(conn)
+            account_id = auth.get_account_id()
+            
+            demo = Demographics(
+                account_id=account_id,
+                gender=request.form.get("gender"),
+                birth_year=int(request.form.get("birthyear")),
+                zip5=request.form.get("zip"),
+                education=request.form.get("education"),
+                race=';'.join(request.form.getlist("race"))
+            )
+
+            repo.store_demographics(demo)
+            conn.commit()
+
+            if auth.get_account_status() == "pending_onboarding_survey":
+                finish_onboarding(account_id)
+                enqueue_newsletter_request(
+                                account_id=account_id,
+                                profile_id=account_id,
+                                group_id=None,
+                                endpoint_url=DEFAULT_RECS_ENDPOINT_URL,
+                            )
+                return redirect(url_for("home", error_description="You have been subscribed!"))
+
+
+
+    today = datetime.date.today()
+    oneyear = timedelta(days=365)
+    yearmin = (today - 100 * oneyear).year # arbitrarilly set to 100 years old
```

Comment:
Let's bump this to 123 years: https://en.wikipedia.org/wiki/List_of_the_verified_oldest_people#:~:text=The%20oldest%20person%20ever%20whose,122%20years%20and%20164%20days.

Reply:
Done.